### PR TITLE
[WFLY-14255] allow configuration of reuse-x-forwarded and rewrite-hos…

### PR DESCRIPTION
**Pull request by baranowb**
…t attributes in ReverseProxy

Issue: https://issues.redhat.com/browse/WFLY-14255

### CODE REVIEW

**Comment by darranl** [comment]
Added "missing-reqs" as a feature request,

**Review by darranl — CHANGES_REQUESTED** [review]
As commented in the other PR, please separate the bump into it's own commit so we can see the changes made to the new version - at the moment the changes specific to this feature are not visible.

File: undertow/src/test/java/org/wildfly/extension/undertow/UndertowSubsystem140TestCase.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2021, Red Hat, Inc., and individual contributors
```
Comment (fl4via) [inline undertow/src/test/java/org/wildfly/extension/undertow/UndertowSubsystem140TestCase.java:3]: Is this really 2021?

File: undertow/src/test/resources/org/wildfly/extension/undertow/undertow-14.0.xml [hunk 2]
```diff
@@ -0,0 +1,126 @@
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
```
Comment (fl4via) [inline undertow/src/test/resources/org/wildfly/extension/undertow/undertow-14.0.xml:3]: Same here. All headers should be checked.

**Review by fl4via — CHANGES_REQUESTED** [review]

**Review by fl4via — APPROVED** [review]

**Comment by fl4via** [comment]
Other than the headers, this looks good to me @baranowb

File: undertow/src/test/java/org/wildfly/extension/undertow/UndertowSubsystem140TestCase.java (hunk 1 above)
Comment (baranowb) [reply undertow/src/test/java/org/wildfly/extension/undertow/UndertowSubsystem140TestCase.java:3]: It was.

**Comment by rezsekzs** [comment]
Hi Guys, 

We are waiting long time ago for that. Because of this missing feature, we have to patch undertow, to override this property's default value under Wildfly. How can we help to speed up the ticket?

Thank So Much

**Comment by baranowb** [comment]
Im going to close this one, as this one is obsolete.

**Pull request closed without merging**